fw/tls.c

```diff
@@ -1010,6 +1012,24 @@ tfw_tls_alpn_match(const TlsCtx *tls, const ttls_alpn_proto *alpn)
 	return false;
 }
 
+static bool
+tfw_ja5t_limit_conn(TlsJa5t fingerprint)
+{
+	u64 limit = tls_get_ja5_conns_limit(fingerprint);
```

Comment:
Is there a sense to even try to calculate conns rate if `limit == U64_MAX`? `ja5t_get_conns_rate` doesn't look cheap.

Reply:
There's no sense